plugin/clone/src/clone_status.cc

```diff
@@ -465,6 +470,34 @@ void Status_pfs::Data::write(bool write_error) {
   status_file.close();
 }
 
+void Status_pfs::Data::write_synchronization_coordinate(
+    const Key_Value &coordinate, bool remove) {
+  std::string file_name;
+  /* Append data directory if cloning to different place. */
+  if (!is_local()) {
+    file_name.assign(m_destination);
+    file_name.append(FN_DIRSEP);
+    file_name.append(CLONE_SYNCHRONIZATION_COORDINATES_FILE);
+  } else {
+    file_name.assign(CLONE_SYNCHRONIZATION_COORDINATES_FILE);
+  }
+  if (remove) {
+    std::filesystem::remove(file_name);
+    return;
+  }
+  m_synchronization_coordinates.push_back(coordinate);
+  std::ofstream status_file;
+  status_file.open(file_name, std::ofstream::out | std::ofstream::trunc);
+  if (!status_file.is_open()) {
+    return;
+  }
+  for (auto &coordinate : m_synchronization_coordinates) {
+    status_file << coordinate.first << std::endl;
```

Comment:
Never use `std::endl`
```suggestion
    status_file << coordinate.first << '\n';
```